src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolderDescriptor.java

```diff
@@ -25,19 +25,44 @@
 package com.cloudbees.hudson.plugins.folder;
 
 import com.cloudbees.hudson.plugins.folder.health.FolderHealthMetricDescriptor;
+import hudson.model.DescriptorVisibilityFilter;
 import hudson.model.TopLevelItemDescriptor;
 import hudson.views.ViewsTabBar;
-import hudson.views.ViewsTabBarDescriptor;
+import java.io.StringWriter;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import jenkins.model.Jenkins;
+import org.apache.commons.jelly.Script;
+import org.apache.commons.jelly.XMLOutput;
+import org.jenkins.ui.icon.IconSpec;
+import org.kohsuke.stapler.MetaClass;
+import org.kohsuke.stapler.Stapler;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.WebApp;
+import org.kohsuke.stapler.jelly.DefaultScriptInvoker;
+import org.kohsuke.stapler.jelly.JellyClassTearOff;
 
 /**
  * Category of {@link AbstractFolder}.
  * @since 4.11-beta-1
  */
-public abstract class AbstractFolderDescriptor extends TopLevelItemDescriptor {
-    
+public abstract class AbstractFolderDescriptor extends TopLevelItemDescriptor implements IconSpec {
+
+    /**
+     * {@inheritDoc}
```

Comment:
You cannot “inherit” documentation on a constructor, because it is not an override.